### DISCUSSION TITLE
Hugo添加waline之后如何打开暗黑模式

### DISCUSSION THREAD

**Discussion by ccfors** [walinejs/waline#1609] [Q&A] [answered]
![image](https://user-images.githubusercontent.com/112556344/203768220-699c059f-7d8b-4aae-96d9-a5bc5f57b860.png)

我的博客是基于Hugo搭建的，而且采用了Wowchemy的模板  https://wowchemy.com/docs/。

由于模板内部结构比较复杂，自己对前端知识了解的比较少。当我在博客文章下面引入waline评论系统之后（仅需要把html文件替换），waline的暗黑模式无法使用。图片中官方教程理解困难，想问下有没有更详细点的设置案例，或者能进一步教一下打开暗黑模式的具体步骤等。

我不需要更多的样式更改，仅仅希望实现暗黑模式，不知道这是否能够在html文件中加入设置或者样式等实现？

我的网站地址是：https://www.ccfors.com/origin/%E7%AC%AC%E4%B8%80%E7%AF%87/

**Comment by Mister-Hope** [comment, marked as answer]
body.dark

**Reply by ccfors** [reply]
能具体讲下在什么地方改不，比如具体哪个或哪类文件，我是真不懂前端额...大佬，求教orz

**Reply by Mister-Hope** [reply]
```js
init({
   // other options
  // ...
  dark: 'body.dark',
});
```

**Reply by ccfors** [reply]
哇，终于解决了，我一直是直接加了body.dark，原来还需要一个单引号。虽然不太理解其中变量类型的转换，但至少解决了。

真的是万分感谢~~
